### PR TITLE
📝 Contributors are provided with a more comprehensive guide on how to begin contributing

### PR DESCRIPTION
**Pull request by zspencer**
https://github.com/zinc-collective/compensated/issues/39

Lane Grant recommended we put together a more concrete guide to getting started
contributing to Compensated. While this does not resolve the technical side of
that (getting CI, making sure bin/setup actually works, etc) it may serve as an
aspirational document that we can start filling in the missing gaps
around?

### CODE REVIEW

File: CONTRIBUTING.md
Comment (user512) [inline CONTRIBUTING.md:8]: TIL about rfc-2119. 😄

**Review by user512 — APPROVED** [review]

**Review by maximegalon5 — APPROVED** [review]
It seems like maintainers have to be technical since they perform regression testing - which is a good thing since it sets up code-pairing to some degree.

I feel as if there should be a 
### Why I should contribute section too. 

This would be similar across other projects to a large degree.

**Comment by zspencer** [comment]
In the spirit of "Contributors Don't Merge Their Own PRs" would someone else be willing to click the big green button?

**Pull request merged**
